api/src/test/java/org/openmrs/util/DatabaseUpdaterDatabaseIT.java

```diff
@@ -29,9 +30,9 @@
 	 * This constant needs to be updated when adding new Liquibase update files to openmrs-core.
 	 */
 	
-	private static final int CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X = 879;
+	private static final int CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X = 880;
 
-	private static final int CHANGE_SET_COUNT_FOR_2_1_X = 870;
```

Comment:
Did you have to change this?

Reply:
Integration test was failing due to increment in changset count.

Reply:
For `CHANGE_SET_COUNT_FOR_2_1_X` or `CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X`?

Reply:
This was for both of them caused by the merge conflicts i had encountered in an earlier PR which i closed.